src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -3912,6 +3926,21 @@ internal static TraceJittedMethod MethodComplete(TraceLoadedDotNetRuntime stats,
             return _method;
         }
 
+        /// <summary>
+        /// Handles AllocRequest event for JIT
+        /// </summary>
+        internal static TraceJittedMethod JitAllocComplete(TraceLoadedDotNetRuntime stats, LoaderHeapAllocRequestTraceData data)
```

Comment:
You can just return `void` here since you don't need to do anything with the method in the caller.